scripts/style_json.js

```diff
@@ -0,0 +1,7 @@
+import { copyFile, writeFile, mkdir } from "fs/promises";
+
+import { buildLayers } from "../src/layer/index.js";
+
+export function generateStyleJSON() {
+  writeFile("dist/style.json", JSON.stringify(buildLayers()));
```

Comment:
This method could call `Label.localizeLayers()`, passing in the contents of [a command line argument](https://nodejs.org/en/knowledge/command-line/how-to-parse-command-line-arguments/) as the locales.